pkg/cmd/codespace/create.go

```diff
@@ -470,6 +495,11 @@ func getMachineName(ctx context.Context, apiClient apiClient, repoID int, machin
 		return machines[0].Name, nil
 	}
 
+	if useWeb {
```

Comment:
In the `useWeb` case we now would only be in this method just to verify that the machine type exists - in that case we would have already returned in the above block because we either matched the machine or we didn't. So I don't think we need this `if` or the `useWeb` parameter anymore

Reply:
Okay will remove the `useWeb` check and `useWeb` param from the `getMachineName` function

Reply:
fixed in [d447e7a](https://github.com/cli/cli/pull/7288/commits/d447e7af91df5be4d6da2f80023784fccd5a13d8)